app/app/settings.py

```diff
@@ -541,8 +544,11 @@
 GITHUB_APP_NAME = env('GITHUB_APP_NAME', default='gitcoin-local')
 
 # Chat
-CHAT_URL = env('CHAT_DRIVER_USER', default='')  # location of where mattermost is hosted
-
+CHAT_PORT = env('CHAT_PORT', default=443)  # port of where mattermost is hosted
+CHAT_URL = env('CHAT_URL', default='https://chat.gitcoin.co')  # location of where mattermost is hosted
```

Comment:
will this default cause any issues when people run the docker containers locally?

Reply:
Yes, will update it to a default for local dev instead.